### PR TITLE
Fix pylint errors

### PR DESCRIPTION
**Pull request by vigneshwaran-nv-10329**

### CODE REVIEW

File: gunicorn/glogging.py [hunk 1]
```diff
@@ -204,9 +204,9 @@ def setup(self, cfg):
             for stream in sys.stdout, sys.stderr:
                 stream.flush()
 
-            self.logfile = open(cfg.errorlog, 'a+')
-            os.dup2(self.logfile.fileno(), sys.stdout.fileno())
-            os.dup2(self.logfile.fileno(), sys.stderr.fileno())
+            with open(cfg.errorlog, 'a+') as self.logfile:
+                os.dup2(self.logfile.fileno(), sys.stdout.fileno())
+                os.dup2(self.logfile.fileno(), sys.stderr.fileno())
```
Comment (isidentical) [inline gunicorn/glogging.py:209]: The problem with this is that, it will set the `self.logfile` to a closed file (since after this context is finalized, it will close the file). So there is no point of setting it to an attribute, since none of the users of that attribute will able to benefit from it.

Comment (benoitc) [reply gunicorn/glogging.py:209]: i don't see the point of using a with statement there.  please keep it that way.

File: gunicorn/workers/gthread.py [hunk 2]
```diff
@@ -49,7 +49,8 @@ def init(self):
         if self.parser is None:
             # wrap the socket if needed
             if self.cfg.is_ssl:
-                self.sock = ssl.wrap_socket(self.sock, server_side=True,
+                context = ssl.create_default_context()
```
Comment (benoitc) [inline gunicorn/workers/gthread.py:52]: this should be a separate commit and not related to pylint really as it is using new code.

**Comment by benoitc** [comment]
not applicable anymore.
